### PR TITLE
make that thing runnable

### PR DESCRIPTION
**Pull request by SiegfriedEhret**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
   "name": "tetrisStack",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "",
   "main": "index.js",
   "scripts": {
+    "install": "node_modules/jspm/jspm.js install",
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "npm run serve",
     "serve": "browser-sync start --server --no-open --files='*.html', src/**/*"
```
Comment (ThomasCrvsr) [inline package.json]: Should it be node_modules/browser-sync/something.js ? like for jspm ?

Comment (SiegfriedEhret) [inline package.json]: Tellemently ! (Tellement in english, you see, right...)

Comment (ThomasCrvsr) [inline package.json]: So much biliguism here !

**Comment by douglasduteil** [comment]
Woops I didn't this lil'PR right there from our friend @SiegfriedEhret

**Comment by douglasduteil** [comment]
I actually partially made the same changes...

**Comment by SiegfriedEhret** [comment]
@douglasduteil amazing !
I didn't know npm was doing that by default, thanks ;)

**Comment by douglasduteil** [comment]
Alright @SiegfriedEhret 

It's 
...
REBASE TIME !

**Comment by SiegfriedEhret** [comment]
@douglasduteil @ThomasCrvsr Paye ta PR nulle...

**Comment by douglasduteil** [comment]
@ThomasCrvsr I stripped him ! I almost done it !
200w 2

**Comment by douglasduteil** [comment]
200w 3

**Comment by douglasduteil** [comment]
Alright :+1:

**Comment by douglasduteil** [comment]
Good job boys

**Pull request merged**